icons/orbit.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="3" />
+  <circle cx="18" cy="6" r="2" />
+  <circle cx="6" cy="18" r="2" />
+  <path d="M18.8,7.8c0.2,0.3,0.4,0.7,0.5,1.1c0.4,1,0.6,2,0.6,3.1s-0.2,2.2-0.6,3.1c-0.4,1-1,1.8-1.7,2.5 s-1.6,1.3-2.5,1.7c-1,0.4-2,0.6-3.1,0.6" />
+  <path d="M5.2,16.2c-0.2-0.3-0.4-0.7-0.5-1.1c-0.4-1-0.6-2-0.6-3.1s0.2-2.2,0.6-3.1c0.4-1,1-1.8,1.7-2.5S7.9,5,8.9,4.6 c1-0.4,2-0.6,3.1-0.6" />
```

Comment:
```suggestion
  <circle cx="5.63" cy="18.37" r="2"/>
  <circle cx="18.36" cy="5.64" r="2"/>
  <circle cx="12" cy="12" r="3"/>
  <path d="M3.05 12.95A9 9 0 0 1 16.8 4.4"/>
  <path d="M20.95 11.04A9 9 0 0 1 7.2 19.6"/>
```

<img width="400" src="http://lucide.dev/api/gh-icon/PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgZmlsbD0ibm9uZSIgc3Ryb2tlPSJjdXJyZW50Q29sb3IiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgc3Ryb2tlLXdpZHRoPSIyIj4KICA8Y2lyY2xlIGN4PSI1LjYzIiBjeT0iMTguMzciIHI9IjIiLz4KICA8Y2lyY2xlIGN4PSIxOC4zNiIgY3k9IjUuNjQiIHI9IjIiLz4KICA8Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSIzIi8+CiAgPHBhdGggZD0iTTMuMDUgMTIuOTVBOSA5IDAgMCAxIDE2LjggNC40Ii8+CiAgPHBhdGggZD0iTTIwLjk1IDExLjA0QTkgOSAwIDAgMSA3LjIgMTkuNiIvPgo8L3N2Zz4=.svg"/>

Reply:
@jguddas: Yeah, definite improvement! Although maybe let's go for pixel perfection instead of regular perfection, e.g.:
```suggestion
  <circle cx="18.5" cy="5.5" r="2" />
  <circle cx="5.5" cy="18.5" r="2" />
  <circle cx="12" cy="12" r="3" />
  <path d="M16.83 4.4A9 9 0 0 0 3.06 13" />
  <path d="M7.17 19.6A9 9 0 0 0 20.94 11" />
```

Reply:
```suggestion
  <circle cx="12" cy="12" r="3" />
  <circle cx="18.5" cy="5.5" r="2" />
  <circle cx="5.5" cy="18.5" r="2" />
  <path d="M16.83 4.4A9 9 0 0 0 3 13" />
  <path d="M7.17 19.6A9 9 0 0 0 21 11" />
```

<img width="400" src="http://lucide.dev/api/gh-icon/PHN2Zz48Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSIzIiAvPjxjaXJjbGUgY3g9IjE4LjUiIGN5PSI1LjUiIHI9IjIiIC8+PGNpcmNsZSBjeD0iNS41IiBjeT0iMTguNSIgcj0iMiIgLz48cGF0aCBkPSJNMTYuODMgNC40QTkgOSAwIDAgMCAzIDEzIiAvPjxwYXRoIGQ9Ik03LjE3IDE5LjZBOSA5IDAgMCAwIDIxIDExIiAvPjwvc3ZnPg==.svg"/>

Reply:
We could increase the radius of the center circle planet element a bit as well.

Reply:
Not sure, that would actually decrease pixel perfection.

Reply:
True, but think about all the bites we save :wink:

Reply:
It's 6. 6 bytes. I think I'd rather go for pixel perfection. 😄